### PR TITLE
Add status variable to PR page

### PR DESCRIPTION
**Pull request by jnewbery**
Add a status {"upcoming", "past"} variable to the meeting page and remove date/time-matching code.

This will allow us to make changes to the page on the day of the meeting without it being moved to the "recent meetings" section.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -86,6 +86,16 @@ Some tips for hosting a great review club meeting:
...
 
+### After the Meeting
+
+One of the review club maintainers will:
+
+- Copy the meeting log to the `##Meeting log` section of the PR page. Meeting
+  logs should be copied exactly, but an `#Erratum` section can be added to
+  correct bad factual errors.
+
+- Change the `status` of the PR page to `past`.
```
Comment (jonatack) [inline CONTRIBUTING.md]: Suggestions:
- `## Meeting Log`
- `## Erratum`
- lines 93 and 97, perhaps replace "PR page" with "post" or "meeting post"

**Comment by jonatack** [comment]
(good idea!)

File: CONTRIBUTING.md (hunk 1 above)
Comment (jnewbery) [reply CONTRIBUTING.md]: fixed!

**Comment by jnewbery** [comment]
Thanks Jon. I've made a couple of small fixes.

**Comment by jonatack** [comment]
Tested ACK.

The CONTRIBUTING.md changes LGTM. Git grepping "Meeting log" shows that all have been changed to "Meeting Log". Git grepping "status:" shows that are all either "past" or "upcoming". The code changes look good.

**Pull request merged**